docs/partials/readmeAddlInfo.md

```diff
@@ -0,0 +1,35 @@
+## Turborepo Overview
+
+This monorepo is managed using [Turborepo](https://turborepo.org/).
+
+### Managing dependencies
+
+#### Best practices for dependency installation
+
+When you install a dependency in a component or package in `auro-formkit`, you should install it directly in the package that uses it. 
+
+The package's `package.json` will have every dependency that it needs. This is true for both external and internal dependencies.
+
+- External dependencies come from the `npm` registry.
+
+- Internal dependencies let you share functionality within your repository.
```

Comment:
I am not sure how those line up to the three types of dependencies in the package JSON (Dependency, Peer Dep, Dev Dep). 

I assume that "external" vs. "internal" is talking about combobox needs input which is internal but input also needs icon and button which is not.

Language should explain this dynamic as other teams are not likely to get it at first.